glossary.yml

```diff
@@ -4247,6 +4247,12 @@
     term: ዲጂታል የነገር መለያ
     def: >
       ለመጽሃፍ ፣ ወረቀት ፣ ሪፖርት ፣ የውሂብ ስብስብ ልዩ ልዩ የማያቋርጥ መለያ የሶፍትዌር መለቀቅ ወይም ሌላ ዲጂታል ቅርሶች።
+  uk:
+    term: "Цифровий ідентифікатор об'єкта"
```

Comment:
Does it have to be upper case?

Reply:
@YehorBoiar you are right - @VeronikaShevc please make lowercase